contracts/colony/ColonyAuthority.sol

```diff
@@ -28,6 +28,8 @@ contract ColonyAuthority is CommonAuthority {
   uint8 constant ARCHITECTURE_ROLE = uint8(ColonyDataTypes.ColonyRole.Architecture);
   uint8 constant ROOT_ROLE = uint8(ColonyDataTypes.ColonyRole.Root);
 
+  // colony is used in the constructor by calls to addRoleCapability, despite what slither thinks
```

Comment:
I don't see where...

Reply:
https://github.com/JoinColony/colonyNetwork/blob/b7201a0b0051ed9a985a0a15e73779b9ced5d780/contracts/colony/ColonyAuthority.sol#L136 - this call in `addRoleCapability` reads the `colony` storage variable.

Reply:
Prior to [solidity 0.8.8](https://github.com/ethereum/solidity/releases/tag/v0.8.8) immutable variables could not be read in the constructor. One to be picked up in #1053

Reply:
Ah, fascinating.